static/styles/input_pill.scss

```diff
@@ -87,7 +87,7 @@
 .pm_recipient .pill-container {
     padding: 0px 2px;
     border: none;
-
+    width: 100%;
```

Comment:
Maybe `flex-grow: 1;` is more appropriate (in the event we add other elements beside the pill container in the future)? This works as well, though.

Reply:
`flex-grow: 1;` seems like a better option :+1: